examples/ios/objc/Migration/Examples/Example_v5.h

```diff
@@ -0,0 +1,239 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#if !defined(SCHEMA_VERSION_0) && !defined(SCHEMA_VERSION_1) && !defined(SCHEMA_VERSION_2) && !defined(SCHEMA_VERSION_3) && !defined(SCHEMA_VERSION_4)
```

Comment:
This will need to be updated to reflect the objc approach to macros I mentioned above